packages/react-reconciler/src/__tests__/ReactBlocks-test.js

```diff
@@ -69,7 +91,7 @@ describe('ReactBlocks', () => {
       );
     }
 
-    let loadUser = block(Query, User);
+    let loadUser = block(User, load);
```

Comment:
Unfortunate that the order gets reversed from the grammar. `User` + `load` = `loadUser`.

Reply:
That's just a naming convention that we could change, though. It's probably helpful to use different names for the result of `block`, since calling `loadUser` here doesn't directly call `load`.